src/XMakeBuildEngine/Definition/ToolsetReader.cs

```diff
@@ -388,6 +394,106 @@ bool accumulateProperties
         }
 
         /// <summary>
+        /// Create a dictionary with standard properties.
+        /// </summary>
+        private static PropertyDictionary<ProjectPropertyInstance> CreateStandardProperties(
+            PropertyDictionary<ProjectPropertyInstance> globalProperties,
+            string version,
+            string root,
+            string toolsPath)
+        {
+#if XPLAT
```

Comment:
Is this something you're adding? Maybe `FEATURE_RUNTIME_AWARENESS` or `FEATURE_MULTIPLE_RUNTIMES`?